apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -120,7 +120,7 @@ function WpcomNux() {
 // Only register plugin if these features are available.
 // If registered without this check, atomic sites without gutenberg enabled will error when loading the editor.
 // These seem to be the only dependencies here that are not supported there.
-if ( __experimentalCreateInterpolateElement && Guide && GuidePage ) {
```

Comment:
You prolly still need to support `__experimentalCreateInterpolateElement` in FSE for older versions of Gutenberg?

Reply:
I'm not sure. On master it worked. With the package update it stopped working. Now with this fix it starts working again. I'm not sure if dependency extraction is doing it for components? Anyways, I'll check on an atomic dev site running an older wp version, thanks for the reminder :)

Reply:
Note that in https://github.com/Automattic/wp-calypso/pull/40828 we get rid of it entirely.

Reply:
if that will be merged soon, I won't do any more investigating then for now :)

Reply:
rebased to include that 🙇